apps/patient/src/views/ReadyBy.tsx

```diff
@@ -82,90 +88,137 @@ export const ReadyBy = () => {
+          </HStack>
+        </Container>
+      </Box>
+
+      <Box pt={5} shadow="inner">
+        <Container pb={selectedTime ? 32 : 8}>
+          <RadioGroup value={`${selectedDay}-${selectedTime}`}>
+            <VStack spacing={3} w="full" align="stretch">
+              {t.readyByOptions[activeTab].map((option) => {
+                const isDisabled = activeTab === 'today' ? checkDisabled(option.label) : false;
 
-          <RadioGroup onChange={setReadyBy} value={readyBy}>
-            <VStack spacing={3} w="full">
-              {t.readyByOptions.map((option) => {
-                const isDisabled = checkDisabled(option.label);
                 return (
-                  <Card
-                    key={option.label}
-                    bgColor={isDisabled ? 'gray.300' : 'white'}
-                    border={isDisabled ? 'gray.300' : '2px solid'}
-                    borderColor={readyBy === option.label ? 'brand.500' : 'white'}
-                    color={isDisabled ? 'gray.600' : 'base'}
-                    onClick={() => !isDisabled && setReadyBy(option.label)}
-                    m="auto"
-                    w="full"
-                    shadow={isDisabled ? 'none' : 'base'}
-                    cursor={isDisabled ? 'not-allowed' : 'pointer'}
-                  >
-                    <CardBody p={3}>
-                      <HStack align="start">
-                        <Radio
-                          mt={1}
-                          value={option.label}
-                          colorScheme="brand"
-                          onClick={(e) => isDisabled && e.preventDefault()}
-                          isDisabled={isDisabled}
-                          cursor={isDisabled ? 'not-allowed' : 'pointer'}
-                        />
-                        <VStack spacing={1}>
-                          <HStack alignSelf="start">
-                            {option.icon ? <Icon as={RxLightningBolt} color="yellow.500" /> : null}
-                            <Text fontWeight="medium">{option.label}</Text>
-                          </HStack>
-                          {option.description ? (
-                            option.badge ? (
-                              <Tag
-                                colorScheme={option.badgeColor}
-                                variant="subtle"
-                                borderRadius="full"
-                              >
-                                {option.description}
-                              </Tag>
-                            ) : (
-                              <Text color="gray.500" fontSize="sm">
-                                {option.description}
-                              </Text>
-                            )
-                          ) : null}
-                        </VStack>
-                      </HStack>
-                    </CardBody>
-                  </Card>
+                  <Fade key={activeTab + '-' + option.label} in={true}>
```

Comment:
SlideFade would be ideal here, but that was a pain to get working. Give it a regular fade to make it clear that you changed tabs.